docs/detections/api/rules/rules-api-bulk-actions.asciidoc

```diff
@@ -581,3 +600,84 @@ If processing of any rule fails, a partial error outputs the ID and/or name of t
 --------------------------------------------------
 
 <1> dev:[] `execution_summary` is under development and its schema may change.
+
+*Example 3, Dry run*
+
+The following request will validate that the `add_index_patterns` bulk action can be successfully applied to three rules. Each rule (which specified by its rule ID) is different: one is a prebuilt Elastic rule, another is a custom machine learning rule, and another is a custom query rule. Because dry run mode is enabled, changes to these rules will not be permanently applied and saved to {es}.
```

Comment:
```suggestion
The following request will validate that the `add_index_patterns` bulk action can be successfully applied to three rules. Each rule (specified by its rule ID) is different: one is a prebuilt Elastic rule, another is a custom machine learning rule, and another is a custom query rule. Because dry run mode is enabled, changes to these rules will not be permanent or saved to {es}.
```